core/src/shred_fetch_stage.rs

```diff
@@ -42,7 +42,7 @@ impl ShredFetchStage {
         repair_context: Option<(&UdpSocket, &ClusterInfo)>,
     ) {
         const STATS_SUBMIT_CADENCE: Duration = Duration::from_secs(1);
-        let mut shreds_received = LruCache::new(DEFAULT_LRU_SIZE);
+        let mut deduper = Deduper::new(DEDUPER_SIZE, DEDUPER_MAX_AGE);
```

Comment:
these two types don't have the same statistical properties. was that fact considered in parameterizing the `Deduper`?

Reply:
Yes.  LRU cache is the wrong data structure.  Once it's saturated it fails catastrophically.  The deduper gradually increases its false positive rate and the false positives are machine diversified.

Reply:
yup, those are the properties i'm talking about.  was more interested in the specific considerations made to translate from the old LRU with 10k entries reset once per slot to the new `Dedupooor` with 16M entries reset every 5min

Reply:
The 5min reset cycle is from the slack discussions.
With the current parameters, 0.001 false positive rate and 76MB bit array, we can consume ~10M _unique_ packets before the false positives go above 0.001 at which point the filter is reset even if we have not hit 5min yet.
If slots are 400ms and ~1000 shreds on average, then 10M would be like ~66 minutes. So normal times we would reset at 5min interval and not sooner.

Reply:
ok, so _way_ more capacity than the old `LRUCache`